opentech/apply/funds/forms.py

```diff
@@ -28,13 +28,8 @@ def clean_action(self):
         if not can_proceed(transition):
             action = self.instance.phase.transitions[action_name]
             raise forms.ValidationError(f'You do not have permission to "{ action }"')
-        self.transition = transition
```

Comment:
Something to consider: I had to move progress_stage to the determination view: https://github.com/OpenTechFund/opentech.fund/pull/396/files#diff-69bf3b7f321569af3a40118621f7c5f3L82